src/components/TodoList/TodoList.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+const TodoList = ({ todos }) => (
+  <ul className='ul'>
+    {todos.map(todo => (
+      <li key={todo.id} className='li'>
```

Comment:
```suggestion
      <li key={todo.id} className="li">
```
Давай в HTML всё таки использовать двойные кавычки